app/config/CookieCryptoFilterProvider.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import com.google.inject.{Inject, Provider}
+import uk.gov.hmrc.crypto.ApplicationCrypto
+import uk.gov.hmrc.play.frontend.filters.{CookieCryptoFilter, SessionCookieCryptoFilter}
+
+class CookieCryptoFilterProvider @Inject()(applicationCrypto: ApplicationCrypto) extends Provider[CookieCryptoFilter] {
```

Comment:
For clarity make it:  
```suggestion
class SessionCookieCryptoFilterProvider @Inject()(applicationCrypto: ApplicationCrypto) extends Provider[CookieCryptoFilter] {
```